bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -250,13 +250,17 @@ spec:
           {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             exec:
-              {{- if not .Values.service.tls.disable_base_client_port }}
-              command: ['/bin/bash', '-c', 'echo "ruok" | timeout {{ .Values.livenessProbe.probeCommandTimeout }} nc -w {{ .Values.livenessProbe.probeCommandTimeout }} localhost {{ .Values.service.port }} | grep imok']
+              {{- if .Values.livenessProbe.useAdmin }}
+              command: ['/bin/bash', '-c', 'curl -s -m {{ .Values.livenessProbe.probeCommandTimeout }} http://localhost:8080/commands/ruok']
               {{- else }}
-                {{- if not .Values.service.tls.client_enable }}
-                command: ['/bin/bash', '-c', 'echo "ruok" | timeout {{ .Values.livenessProbe.probeCommandTimeout }} openssl s_client -quiet -crlf -connect localhost:{{ .Values.service.tls.client_port }} | grep imok']
+                {{- if not .Values.service.tls.disable_base_client_port }}
+                command: ['/bin/bash', '-c', 'echo "ruok" | timeout {{ .Values.livenessProbe.probeCommandTimeout }} nc -w {{ .Values.livenessProbe.probeCommandTimeout }} localhost {{ .Values.service.port }} | grep imok']
                 {{- else }}
-                command: ['/bin/bash', '-c', 'echo "ruok" | timeout {{ .Values.livenessProbe.probeCommandTimeout }} openssl s_client -quiet -crlf -connect localhost:{{ .Values.service.tls.client_port }} -cert {{ .Values.service.tls.client_cert_pem_path }} -key {{ .Values.service.tls.client_key_pem_path }} | grep imok']
+                  {{- if not .Values.service.tls.client_enable }}
+                  command: ['/bin/bash', '-c', 'echo "ruok" | timeout {{ .Values.livenessProbe.probeCommandTimeout }} openssl s_client -quiet -crlf -connect localhost:{{ .Values.service.tls.client_port }} | grep imok']
+                  {{- else }}
+                  command: ['/bin/bash', '-c', 'echo "ruok" | timeout {{ .Values.livenessProbe.probeCommandTimeout }} openssl s_client -quiet -crlf -connect localhost:{{ .Values.service.tls.client_port }} -cert {{ .Values.service.tls.client_cert_pem_path }} -key {{ .Values.service.tls.client_key_pem_path }} | grep imok']
+                  {{- end }}
```

Comment:
Why is this block indented two extra spaces?

Reply:
Looks like an oversight on my part, will change it.

Reply:
Actually not an oversight - I'm not sure the right convention to use here.

It looks like each nested if/else in the original chart added a layer of indentation. Since I added a new one with `{{- if .Values.livenessProbe.useAdmin }}`, I added another two spaces of indentation. Should I strip that out?

Reply:
Yes strip that out, otherwise the YAML won't be correctly formatted.